### ISSUE TITLE
Test suite build failure with GHC 8.2

### ISSUE THREAD

**Issue by snoyberg** [bos/attoparsec#131]
```
Building test suite 'tests' for attoparsec-0.13.1.0..

<no location info>: warning: [-Wmissing-home-modules]
    These modules are needed for compilation but not listed in your .cabal file's other-modules: Data.Attoparsec.ByteString
                                                                                                 Data.Attoparsec.ByteString.Buffer
                                                                                                 Data.Attoparsec.ByteString.Char8
                                                                                                 Data.Attoparsec.ByteString.FastSet
                                                                                                 Data.Attoparsec.ByteString.Internal
                                                                                                 Data.Attoparsec.ByteString.Lazy
                                                                                                 Data.Attoparsec.Combinator
                                                                                                 Data.Attoparsec.Internal
                                                                                                 Data.Attoparsec.Internal.Fhthagn
                                                                                                 Data.Attoparsec.Internal.Types
                                                                                                 Data.Attoparsec.Number
                                                                                                 Data.Attoparsec.Text
                                                                                                 Data.Attoparsec.Text.Buffer
                                                                                                 Data.Attoparsec.Text.FastSet
                                                                                                 Data.Attoparsec.Text.Internal
                                                                                                 Data.Attoparsec.Text.Lazy
                                                                                                 Data.Attoparsec.Zepto
[ 1 of 29] Compiling Data.Attoparsec.ByteString.FastSet ( Data/Attoparsec/ByteString/FastSet.hs, dist/build/tests/tests-tmp/Data/Attoparsec/ByteString/FastSet.o )
[ 2 of 29] Compiling Data.Attoparsec.Internal.Fhthagn ( Data/Attoparsec/Internal/Fhthagn.hs, dist/build/tests/tests-tmp/Data/Attoparsec/Internal/Fhthagn.o )
```

**Comment by bgamari** [comment]
Thanks @snoyberg!

**Comment by hvr** [comment]
This is most likely caused by some missing upper bound.

**Comment by bgamari** [comment]
Released in 0.13.2.0.
